deploy/bundle/manifests/controller.devfile.io_devworkspaceoperatorconfigs.yaml

```diff
@@ -80,6 +80,1278 @@ spec:
                         pattern: ^(\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))(([KMGTPE]i)|[numkMGTPE]|([eE](\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))))?$
                         x-kubernetes-int-or-string: true
                     type: object
+                  defaultTemplate:
```

Comment:
@AObuchow could you please clarify if the spec for `defaultTemplate` was created from scratch?

Reply:
It was automatically created with `make generate manifests fmt generate_default_deployment generate_olm_bundle_yaml`. The reason it is so large is because of all the fields contained within the `DevWorkspaceTemplateSpecContent` struct